samples/manifests/demo/lake-admin-plugins.yaml

```diff
@@ -25,9 +25,17 @@
         image: public.ecr.aws/v3o4w1g6/aws-orbit-workbench/utility-data:latest
         uid: 0
         gid: 0
--   PluginId: overprovisioning1
+-   PluginId: overprovisioning-primary-compute
```

Comment:
is the update to overprovisioning in this PR intentional? or did the branch just need updated?

Reply:
this is already in the main branch, so no impact